### DISCUSSION TITLE
ScandiPWA support for Magento v 2.4.5+

### DISCUSSION THREAD

**Discussion by nnirmana** [scandipwa/scandipwa#5416] [General]
According to this URL https://manual.scandipwa.com/pwa/magento-version-mapping , SPWA only supports up to 2.4.4
Does anyone when we will be getting 2.4.5 support ? and what Magento version will be supported by version 6 ?

**Comment by rsidhaarth** [comment]
Version 6.2.0 will support 2.4.6

**Comment by nnirmana** [comment]
@rsidhaarth Thanks for the reply, Do you know when they are planning to release v 6.2.0 ?

**Comment by jbruni** [comment]
I would like to see the "Magento version mapping" page updated with an entry for Magento version 2.4.7  <-> ScandiPWA version 6.4.0+ 

https://manual.scandipwa.com/pwa/magento-version-mapping

But I could not find a repository in GitHub to create a pull request for this documentation update.
